docker-app/qfieldcloud/core/admin.py

```diff
@@ -787,6 +788,17 @@ def get_inlines(self, request, obj=None):
 
         return inlines
 
+    def get_urls(self):
+        urls = super().get_urls()
+        return [
+            path(
+                "/<uuid:apply_job_id>/export-deltafile",
+                self.admin_site.admin_view(self.export_applyjob_deltafile),
+                name="export_applyjob_deltafile",
+            ),
+            *urls,
+        ]
+
```

Comment:
Maybe we could somehow declare a specific `class ApplyJobAdmin(JobAdmin)` which adds this functionality and also the two previous methods where we have to check for `if obj and obj.type == Job.Type.DELTA_APPLY` and still show them in one job-list-view? Would be nice but not necessary if hard to  implement.

Reply:
Been there before, it's harder to maintain when we have only 3 job types. I would prefer to keep it as it is for now.

Reply:
sure - I actually thought you might have considered this before ;)